app/components/job/job-hook-task/job-hook-task-details/job-hook-task-details.html

```diff
@@ -1,9 +1,10 @@
-    <div *ngIf="task">
+<div *ngIf="task">
     <bl-property-group>
         <bl-link-property label="Pool" [value]="task.poolId" [link]="['/pools', task.poolId]"></bl-link-property>
         <bl-link-property label="Node" [value]="task.nodeId" [link]="['/pools', task.poolId, 'nodes', task.nodeId]"></bl-link-property>
-
-        <bl-node-file-list #list [poolId]="task.poolId" [nodeId]="task.nodeId" [folder]="currentFolder" *ngIf="currentFolder">
-        </bl-node-file-list>
+        Cur: {{currentFolder}}
```

Comment:
should this be "Current"? or is "Cur" ok here?

Reply:
nope should not be there, debugging, good catch